plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -1819,7 +1826,7 @@ protected void testTargetMaxFileSize(int expectedTableWriters)
 
```

Comment:
Why didn't the tests related to `testTargetMaxFileSize` start failing ? Shouldn't task writer scaling have failed the expected writer counts assertions there ?

Reply:
physical written bytes never crosses the 32MB limit

Reply:
Okay, let's disable task writer scaling in those tests to avoid any flakiness in future

Reply:
already done